src/Tools/Source/CompilerGeneratorTools/Source/CSharpErrorFactsGenerator/ErrorGenerator.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// We only build the Source Generator in the netstandard target
+#if NETSTANDARD
+
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.CodeAnalysis.CSharp.Internal.CSharpErrorFactsGenerator
```

Comment:
The namespace here is part of the generated file path. I think I should shorten it to only `CSharpErrorFactsGenerator`, does that sound good?

Reply:
Sure. That would match CSharpSyntaxGenerator. The `<RootNamespace>` would also need to be updated in the project file.